framework_tvb/tvb/interfaces/web/static/js/bursts.js

```diff
@@ -420,11 +420,20 @@ function _displayPseSimulationMessage() {
     const THREASHOLD_WARNING = 500;
     const THREASHOLD_ERROR = 50000;
 
-    pse_param1_number = _computeRangeNumberForParamPrefix('pse_param1');
-    pse_param2_number = _computeRangeNumberForParamPrefix('pse_param2');
+    var maxRangeNumber = document.getElementById("max_range_number");
+    var maxRangeNumberInt = parseInt(maxRangeNumber.value);
+
+    let pse_param1_number = _computeRangeNumberForParamPrefix('pse_param1');
+    let pse_param2_number = _computeRangeNumberForParamPrefix('pse_param2');
 
     let nrOps = pse_param1_number * pse_param2_number;
 
+    if (nrOps > maxRangeNumberInt){
+        message = "Can't launch PSE! Maximum number of operations has been exceeded!";
```

Comment:
Maybe inform the user what is the maximum nr of operations, to have an idea on how to configure the PSE correctly